validphys2/src/validphys/filters.py

```diff
@@ -9,13 +9,39 @@
 
 import numpy as np
 
-from NNPDF import CommonData
 from reportengine.checks import make_argcheck, check, check_positive, make_check
 from reportengine.compat import yaml
 import validphys.cuts
 
 log = logging.getLogger(__name__)
 
+KIN_LABEL = {
```

Comment:
I guess the plural would be more consistent with the content of `commondataparser.py` (btw it is also not separating by underscore - both fine, but I would pick just one).